src/compiler/checker.ts

```diff
@@ -42550,6 +42568,21 @@ namespace ts {
             return false;
         }
 
+        function checkGrammarImportCallArguments(node: ImportCall, nodeArguments: NodeArray<Expression>): boolean {
+            if (moduleKind !== ModuleKind.ESNext) {
+                checkGrammarForDisallowedTrailingComma(nodeArguments);

```

Comment:
Maybe you should leave a comment that, after ESNext, trailing comma is allowed in dynamic import, otherwise this looks like a typo to me at the first glance.

Reply:
Okay